utils/vibe/utils/dictionarylist.d

```diff
@@ -400,3 +400,15 @@ unittest {
 	assert(text(l) == `["foo": 42, "bar": 43]`, text(l));
 	assert(l.toString() == `["foo": 42, "bar": 43]`, l.toString());
 }
+
+// Issue 2004
+@safe unittest {
```

Comment:
I think the failures are coming from `Variant` not being annotated `@trusted`. Since this is just about the `inout` issue, I'll just quickly remove the `@safe`.